translator/jsonconfig/mergeJsonConfig.go

```diff
@@ -16,20 +17,29 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/translator/util"
 )
 
-func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string) (map[string]interface{}, error) {
+func MergeJsonConfigMaps(jsonConfigMapMap map[string]map[string]interface{}, defaultJsonConfigMap map[string]interface{}, multiConfig string, strictValidation bool, inputJsonDirPathErr error ) (map[string]interface{}, error) {
```

Comment:
Personally, I don't like the idea of passing in an `error` as an input to a function.

Reply:
Agree. Passing a boolean instead of `error`. as an input to a function.